phpBB/styles/prosilver/template/display_options.html

```diff
@@ -13,10 +14,10 @@
 				<hr class="dashed" />
 				<input type="submit" class="button1 button button-form-bold" name="sort" value="{L_SORT}" />
 			<!-- ELSE -->
-				<label>{L_DISPLAY}{L_COLON} {S_SELECT_SORT_DAYS}</label>
+				<label>{L_DISPLAY}{L_COLON} {% if S_SELECT_SORT_DAYS is iterable %}{% with { select: S_SELECT_SORT_DAYS } only %}{{ block('select_form_widget') }}{% endwith %}{% else %}{S_SELECT_SORT_DAYS}{% endif %}</label>
```

Comment:
I understand that this file is still using phpBB syntax, but seeing you're refactoring these lines anyway might aswell use TWIG syntax. Or atleast for the `{{ S_SELECT_SORT_DAYS }}`.